README.md

```diff
@@ -16,7 +16,7 @@
 
 The client is written in Go, with pre-compiled binaries available for Mac,
 Windows, Linux, and FreeBSD. Check out the [website](http://git-lfs.github.com)
-for an overview of features.
+for an overview of all features.
```

Comment:
Could we say "more" here instead of "all"? I don't think that the website is entirely comprehensive, so "more" might be a more accurate way to get the message across.